src/fast_array_utils/stats/_sum.py

```diff
@@ -69,39 +96,73 @@ def _sum_dask(
         msg = "sum does not support numpy matrices"
         raise TypeError(msg)
 
-    def sum_drop_keepdims(
-        a: CpuArray,
+    def sum_dask_inner(
+        a: CpuArray | GpuArray,
         /,
         *,
-        axis: tuple[Literal[0], Literal[1]] | Literal[0, 1, None] = None,
+        axis: ComplexAxis = None,
         dtype: DTypeLike | None = None,
         keepdims: bool = False,
     ) -> NDArray[Any] | types.CupyArray:
-        del keepdims
-        if a.ndim == 1:
```

Comment:
The problem was that if someone passed in `axis=0` previously, this would not have errored?

Reply:
Right, yup